### PR TITLE
Update MySQL Troubleshooting with New Relic

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes: #5902 

## Summary

**[MySQL Troubleshooting with New Relic APM](https://pantheon.io/docs/debug-mysql-new-relic/)** - Small update to reflect changes in the New Relic APM user interface. 

## Effect
- Adjustment for UI change
- Markdown Linting and review date

## Post Launch
none

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6039](https://pr-6039--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `65` | `55.00000000000001` | `-10` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-6039` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/24129/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/24129/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6039--pantheon-docs.my.pantheonfrontend.website/docs)

File: source/content/debug-mysql-new-relic.md [hunk 1]
```diff
@@ -48,9 +56,15 @@ Now that the problem has been found, it can be addressed. In this case, simply a
...
+
 1. Within those transactions, go into the SQL trace to discover long running queries.
+
 1. Using SFTP, download the appropriate MySQL Slow Log to retrieve the query in its entirety.
+
 1. Connect to a safe MySQL server via CLI. Run the query to test the performance.
+
 1. If the query result is poor, use the `EXPLAIN` and `EXPLAIN EXTENDED` MySQL command to get additional information. You can also examine the MySQL tables for structural issues using `DESCRIBE` and `ANALYZE` commands.
+
 1. Once identified, fix the issue. This can be within the MySQL server itself if that's where the problem is, or it can be within the application by redoing code or configurations that are creating the errant query.
```
Comment (marfillaster) [inline source/content/debug-mysql-new-relic.md:70]: `within MySQL server itself` implies server reconfiguration which is not feasible. Maybe `the SQL query`

Comment (alexfornuto) [reply source/content/debug-mysql-new-relic.md]: @marfillaster how does https://github.com/pantheon-systems/documentation/pull/6039 grab ya?

Comment (marfillaster) [reply source/content/debug-mysql-new-relic.md]: Looks good

**Review by EdwardAngert — APPROVED** [review]

**Pull request merged**